spec/factories/check.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :check do
+    status { :approved }
```

Comment:
Shouldn't this be in an `approved` trait?

Reply:
Yeah, but that'll come later. At the moment, the only status we can have is `approved`

Reply:
If we do it now, it reduces the thrashing on specs when we do add it, making the history cleaner. Adding `rejected` shouldn't men changes to `approved`, imo.

Reply:
For me this was about catching the right balance of small steps vs knowing what is coming. As this work was about a check that could only result in an approval, we purposely only took steps to make that possible. 

Funnily enough, we spoke about how to best strike that balance as it's something I struggle with.